src/assets/stylesheets/_variables.scss

```diff
@@ -63,9 +63,13 @@ $chartColors: $chart0 $chart1 $chart2 $chart3 $chart4 $chart5 $chart6;
 $sliderTrackFillRed: red;
 $sliderTrackFillBlue: blue;
 $sliderTrackFillGreen: green;
+$sliderTrackFillPurple: violet;
+$sliderTrackFillCyan: cyan;
+$sliderTrackFillYellow: yellow;
+$sliderTrackFillOrange: orange;
```

Comment:
There might be a discussion later about the specific hex codes that will be used, but this is certainly great start and personally, I think these do just fine 👍